src/components/FullWidthPromo/_full-width-promo.scss

```diff
@@ -0,0 +1,165 @@
+// ----------------------------------
+// UI Components
+// Full Width Promo
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+.cc-fw-promo {
+  background: var(--colour-amber-05);
+  position: relative;
+}
+
+.cc-fw-promo__image {
+  @extend %img-cover;
+  margin: 0;
+  
+  @include mq($until: sm) {
+    height: calc(220vw / 320 * 100);
+  }
+
+  @include mq(sm) {
+    bottom: 0;
+    left: 0;
+    position: absolute;
+    right: 0;
+    top: 0;
+    z-index: 0;
+  }
+}
+
+.cc-fw-promo__container {
+  @include grid-container;
+  padding: calc(7.5 * var(--space-unit)) 0;
```

Comment:
I think the shorthand may be detrimental here? Seems like we only want to set top/bottom whereas this explicitly sets left/right to 0 as well, which:

- adds unnecessary style declaration
- makes it harder to overwrite (depending on another selectors specificity/order)